### PR TITLE
fix: ensure api-spec not globally modified

### PR DESCRIPTION
**Pull request by nimerritt**
Swaggerize-hapi
(https://github.com/krakenjs/swaggerize-hapi/blob/2c16d3379afc7b0a4941247881061b06b6d8a750/lib/index.js#L29)
modified the API spec by adding a `/` as a default base path. This
breaks our matching and is inconsistent with the spec.

### CODE REVIEW

File: src/static-api-server.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ import preResponse from './extension-points/pre-response';
 import { createValidator, ValidationError } from '@springworks/m2h-schema-validator';
 import * as hapi_sov from '@springworks/hapi-sov';
 
+function deepClone(obj) {
+  return JSON.parse(JSON.stringify(obj));
+}
+
```
Comment (jishi) [inline src/static-api-server.js]: :nauseated_face: But in this case it is probably fine. :)

**Comment by jishi** [comment]
But this still don't fix when someone actually enters basePath: '/' explicitly.

File: src/static-api-server.js (hunk 1 above)
Comment (nimerritt) [inline src/static-api-server.js]: I know, but I don't want lodash.

Comment (jishi) [inline src/static-api-server.js]: You already have lodash as dependency here though :)

**Pull request merged**